tests/conftest.py

```diff
@@ -212,6 +212,8 @@
 from ocs_ci.helpers.keyrotation_helper import PVKeyrotation
 from ocs_ci.ocs.resources.storage_cluster import set_in_transit_encryption
 from ocs_ci.helpers.e2e_helpers import verify_osd_used_capacity_greater_than_expected
+from ocp_resources.resource import get_client
```

Comment:
This import here is causing the pr_checks (3.9) failure:
```
  >       return self._fmt % record.__dict__
  E       KeyError: 'clusterctx'
```
Please move the import directly to the `admin_client` fixture, which should fix the issue.

Reply:
Moved from ocp_resources.resource import get_client to admin client fixture.